external_deps/build.sh

```diff
@@ -863,6 +863,9 @@ if [ "${#}" -lt "2" ]; then
 	Linux arm64 and armhf native compile:
 	  naclsdk (and possibly others depending on what packages your distribution provides)
 
+	Packages also required by Linux platforms when building a release:
```

Comment:
I'd say something like "Packages required by the Unvanquished release script for Linux" and list all of them (without subtracting nacl)

Reply:
> (without subtracting nacl)

The arm linux packages don't ship naclports and naclports is not used by the Unvanquished release script.

That's why I used the wording `also required by Linux platforms`

Reply:
I mean that naclsdk should be included. The runtime binaries are required.

I think the Unvanquished release script should be mentioned specifically because this is just a random set of packages where using the one from the Debian repositories wasn't good for whatever reason. If someone made a release script for a different platform the list could be different.

Reply:
This sentence will be totally rewritten the day we are able to build the release for all platforms with the same libraries. I removed it, there is no need to spend time on that.